x/crosschain/types/message_remove_inbound_tracker.go

```diff
@@ -0,0 +1,51 @@
+package types
+
+import (
+	errorsmod "cosmossdk.io/errors"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+)
+
+const TypeMsgRemoveInboundTracker = "RemoveInboundTracker"
+
+var _ sdk.Msg = &MsgRemoveInboundTracker{}
+
+func NewMsgRemoveInboundTracker(creator string, chain int64, txHash string) *MsgRemoveInboundTracker {
+	return &MsgRemoveInboundTracker{
+		Creator: creator,
+		ChainId: chain,
+		TxHash:  txHash,
+	}
+}
+
+func (msg *MsgRemoveInboundTracker) Route() string {
+	return RouterKey
+}
+
+func (msg *MsgRemoveInboundTracker) Type() string {
+	return TypeMsgRemoveInboundTracker
+}
+
+func (msg *MsgRemoveInboundTracker) GetSigners() []sdk.AccAddress {
+	creator, err := sdk.AccAddressFromBech32(msg.Creator)
+	if err != nil {
+		panic(err)
+	}
+	return []sdk.AccAddress{creator}
+}
+
+func (msg *MsgRemoveInboundTracker) GetSignBytes() []byte {
+	bz := ModuleCdc.MustMarshalJSON(msg)
+	return sdk.MustSortJSON(bz)
+}
+
+func (msg *MsgRemoveInboundTracker) ValidateBasic() error {
```

Comment:
I think its okay to be be not stringent about validations 
- This message is sent by policy address 
- If the hash or chainID is wrong, it does nothing